Neos.Flow/Classes/Error/Debugger.php

```diff
@@ -466,8 +466,11 @@ public static function getCodeSnippet(string $filePathAndName, int $lineNumber,
 
     protected static function getObjectSnippetPlaintext(object $object): string
     {
-        if (is_callable([$object, '__toString'])) {
-            return self::getObjectShortName($object) . '|' . self::truncateObjectOutput((string)$object) . '|';
+        if (method_exists($object, '__toString')) {
+            try {
+                $string = (string)$object;
+                return self::getObjectShortName($object) . '|' . self::truncateObjectOutput($string) . '|';
+            } catch (\Throwable $_) {/* This can happen if what was callable was not actually __toString (a magic __call() will do that) we can try other ways to get information. */}
```

Comment:
```suggestion
            } catch (\Throwable $_) {/* This can happen if what was callable was not actually __toString (a magic __call() will do that) we can try other ways to get information. */
            }
```